woocommerce/woocommerce
# Add product tags data store

### All Submissions:






### Changes proposed in this Pull Request:



Closes #33467, closes #33468, closes #33469 .

### How to test the changes in this Pull Request:

1. Attempt to get product tags `wp.data.select('wc/admin/products/tags').getProductTags({});`. Note that you. may have to run this again after the fetch has finished resolving.
2. Optionally, pass some query parameters found in https://woocommerce.github.io/woocommerce-rest-api-docs/#list-all-product-tags
`wp.data.select('wc/admin/products/tags').getProductTags({page: 2});`
3. Using one of the tag IDs, update some tag properties - `wp.data.dispatch('wc/admin/products/tags').updateProductTag(166, { name: 'Taggalicious'  } );`
4. Check that the tag was updated
5. Attempt to create a new tag - `wp.data.dispatch('wc/admin/products/tags').createProductTag({ name: 'Sir Tagsalot'  } );`
6. Check that the tag was added
7. Attempt to delete a tag - `wp.data.dispatch('wc/admin/products/tags').deleteProductTag(166);`
8. Check that the tag was deleted


### Other information:




### FOR PR REVIEWER ONLY:
